### PR TITLE
usage restriction fixes

### PR DESCRIPTION
**Pull request by NikitaPoly**

### CODE REVIEW

File: deployments/pulse3d/services/pulse3d/src/models/models.py [hunk 1]
```diff
@@ -78,5 +78,5 @@ class UploadDownloadRequest(BaseModel):
 
 
 class GenericErrorResponse(BaseModel):
-    message: Union[str, Dict[str, bool]]
+    message: UsageQuota
```
Comment (luciipak) [inline deployments/pulse3d/services/pulse3d/src/models/models.py:81]: This needs to keep the string option,  `message:  Union[str, UsageQuota]`

Comment (luciipak) [reply deployments/pulse3d/services/pulse3d/src/models/models.py:81]: or `str | ...`

File: frontend/pulse3d/src/components/basicWidgets/CircularProgressWithLabel.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import Typography from "@mui/material/Typography";
 export default function CircularProgressWithLabel(props) {
   return (
     <Box sx={{ position: "relative", display: "inline-flex" }}>
-      <CircularProgress variant="determinate" {...props} style={{ color: "var(--teal-green)" }} />
+      <CircularProgress variant="determinate" value={props.value} style={{ color: "var(--teal-green)" }} />
```
Comment (luciipak) [inline frontend/pulse3d/src/components/basicWidgets/CircularProgressWithLabel.js:7]: why was this necessary?

**Comment by luciipak** [comment]
Did you fix the issue with the credit usage warning modal not popping up with  interactive analysis?

File: frontend/pulse3d/src/components/basicWidgets/CircularProgressWithLabel.js (hunk 2 above)
Comment (NikitaPoly) [reply frontend/pulse3d/src/components/basicWidgets/CircularProgressWithLabel.js:7]: This removed an annoying error message from the console. It is not present in production but fires off every page load when testing.

Comment (luciipak) [reply frontend/pulse3d/src/components/basicWidgets/CircularProgressWithLabel.js:7]: You can also set default props and do `CircularProgressWithLabel({ value, colorOfTextLabel = "black" })` so remove the extra code and conditional

**Comment by luciipak** [comment]
These need to be merged into 02.16.23 also
